fili-core/src/main/java/com/yahoo/bard/webservice/web/ApiFilter.java

```diff
@@ -142,6 +149,15 @@ public FilterOperation getOperation() {
      * @return the set of values
      */
     public Set<String> getValues() {
+        return new LinkedHashSet<>(this.values);
+    }
+
+    /**
+     * Getter for this ApiFilter's set of filter values.
+     *
+     * @return the list of values
+     */
+    public List<String> getValuesList() {
```

Comment:
I don't know why we have 2 getters. Could we simply remove this?

Reply:
No @archolewa wants backward compatibility for existing getter methods